main.go

```diff
@@ -77,6 +80,390 @@ func main() {
 					return nil
 				},
 			},
+			{
+				Name:  "create-customer",
+				Usage: "Creates a new customer to go in the customers database, must specify email and state(2 letter code)",
+				Action: func(cCtx *cli.Context) error {
+					if len(os.Args) != 4 {
+						return errors.New("Must specify email and state")
+					}
+					email := os.Args[2]
+					state := os.Args[3]
+					insertStatement := "INSERT INTO Customers (email, state) VALUES (?, ?)"
+					_, err := db.Exec(insertStatement, email, state)
+					if err != nil {
+						log.Fatal(err)
+					}
+					idQuery := "SELECT LAST_INSERT_ID()"
+					var customerID int
+					err = db.QueryRow(idQuery).Scan(&customerID)
+					if err != nil {
+						log.Fatal(err)
+					}
+					fmt.Println("CUSTOMER_ID\tEMAIL\t\t\tSTATE")
+					fmt.Printf("%d\t\t%s\t%s\n", customerID, email, state)
+
+					return nil
+				},
+			},
+			{
+				Name:  "create-product",
+				Usage: "Creates a new product to go in the products database, must specify name and price",
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:  "sku",
+						Usage: "the sku of the product",
+					},
+				},
+				Action: func(cCtx *cli.Context) error {
+					if cCtx.NArg() < 2 {
+						return errors.New("Must specify name and price")
+					}
+
+					name := cCtx.Args().Get(0)
+					price, err1 := strconv.Atoi(cCtx.Args().Get(1))
+					if err1 != nil {
+						log.Fatal(err1)
+					}
+					sku := cCtx.String("sku")
+					if sku != "" {
+						insertStatement := "INSERT INTO Products (name, price, sku) VALUES (?, ?, ?)"
+						_, err := db.Exec(insertStatement, name, price, sku)
+						if err != nil {
+							log.Fatal(err)
+						}
+						idQuery := "SELECT LAST_INSERT_ID()"
+						var productId int
+						err = db.QueryRow(idQuery).Scan(&productId)
+						if err != nil {
+							log.Fatal(err)
+						}
+						fmt.Println("Product_ID\tName\t\t\tPrice\t\t\tSku")
+						fmt.Printf("%d\t\t%s\t%v\t%s\n", productId, name, price, sku)
+					} else {
+						insertStatement := "INSERT INTO Products (name, price) VALUES (?, ?)"
+						_, err := db.Exec(insertStatement, name, price)
+						if err != nil {
+							log.Fatal(err)
+						}
+						idQuery := "SELECT LAST_INSERT_ID()"
+						var productId int
+						err = db.QueryRow(idQuery).Scan(&productId)
+						if err != nil {
+							log.Fatal(err)
+						}
+						fmt.Println("PRODUCT_ID\tNAME\t\t\tPRICE")
+						fmt.Printf("%d\t\t%s\t%v\n", productId, name, price)
+					}
+					return nil
+
+				},
+			},
+			{
+				Name:  "create-order",
+				Usage: "Creates a new order to go in the order database, must specify customer_id and product_id",
+				Action: func(cCtx *cli.Context) error {
+					if len(os.Args) != 4 {
+						return errors.New("Must specify product_id and customer_id")
+					}
+					pId, error1 := strconv.Atoi(os.Args[2])
+					cId, err1 := strconv.Atoi(os.Args[3])
+					if error1 != nil {
+						log.Fatal(error1)
+					}
+					if err1 != nil {
+						log.Fatal(err1)
+					}
```

Comment:
You always want to check your error as soon as you assign it (also, it's preferred to re-use errors instead of creating new variables)
```suggestion
					pId, err := strconv.Atoi(os.Args[2])
					if err != nil {
						return err 
					}
					cId, err := strconv.Atoi(os.Args[3])
					if err != nil {
						return err 
					}
```